frontend/src/layouts/AppLayout/AppLayout.tsx

```diff
@@ -423,7 +426,7 @@ export const AppLayout = ({ children }: LayoutProps) => {
                       <Select
                         defaultValue={currentWorkspace?.id}
                         value={currentWorkspace?.id}
-                        className="w-full truncate bg-mineshaft-600 py-2.5 font-medium"
+                        className="w-full [&>*:first-child]:truncate bg-mineshaft-600 py-2.5 font-medium"
```

Comment:
We can also add as `first:` pseudo-class variants instead of `[&>*:first-child]:` rfc which one the best

[docs](https://v1.tailwindcss.com/docs/pseudo-class-variants#first-child)

Reply:
Ah... can't use first: as we can't access the child component and `first-child` is not a thing in tailwind. So, this is our best options or we can try update add the class in the primitive select component.